ember_debug/general-debug.js

```diff
@@ -31,27 +30,27 @@ var GeneralDebug = Ember.Object.extend(PortMixin, {
       }
     });
     return config;
-  }).property(),
+  }),
 
 
-  sendBooted: function() {
+  sendBooted() {
     this.sendMessage('applicationBooted', {
       booted: this.get('application.__inspector__booted')
     });
   },
 
-  sendReset: function() {
+  sendReset() {
     this.sendMessage('reset', {
       reset: true
     });
   },
 
   messages: {
-    applicationBooted: function() {
+    applicationBooted() {
       this.sendBooted();
     },
-    getLibraries: function() {
-      var libraries = Ember.libraries;
+    getLibraries() {
+      let libraries = Ember.libraries;
```

Comment:
Is it possible to import `libraries` as a const at the top of the file?
`const { libraries } = Ember`